src/_includes/cloud/enable-ssh.md

```diff
@@ -165,6 +166,19 @@ To add an SSH key using the Project Web Interface:
 
 1. Follow the prompts on your screen to complete the task.
 
+### Add a key using the Cloud Account Dashboard {#add-key-cloud}
+
+You will add your SSH public key to your Cloud account.
+
+1. Go to https://accounts.magento.cloud
+1. Click on 'Account Settings'.
+1. Under SSH keys, click on 'Add a public key'.
+1. Have another user on the project (with the Contributor role - at minimum) to redeploy each active environment you will have access to.
```

Comment:
```suggestion
1. Have another user on the project (with the Contributor role - at minimum) to redeploy each active environment you have access to. This will upload your SSH key to the environment.
```